test/info.py

```diff
@@ -101,6 +102,7 @@
     (   0x1017,     'lpc11u35_hrm1017_if',                      None,               'mbed-HRM1017'                          ),
     (   0x1018,     'lpc11u35_ssci824_if',                      None,               'Switch-Science-mbed-LPC824'            ),
     (   0x1019,     'lpc11u35_sscity_if',                       None,               'Switch-Science-mbed-TY51822r3'         ),
+    (   0x1022,     'lpc11u35_ro359b_if',                       None,               'ro359b'                                ),
```

Comment:
Hi @c1728p9 , is [this](https://github.com/mbedmicro/DAPLink/pull/266/commits/040e5a2b1141934a630b612ce325f37c8298a7be) right fix?